mobile/client.go

```diff
@@ -163,8 +170,29 @@ func (os *Objects) Get(i int) *Props { return os.a[i] }
 func (os *Objects) Length() int      { return len(os.a) }
 func (os *Objects) String() string   { return fmt.Sprint(os.a) }
 
+type PropsUtil struct {
+}
```

Comment:
Do the string-bytearray conversions in the Props methods insted, if the conversion is necessary.

Reply:
Why is PropsUtil still here?

Reply:
Gotcha, I was also under the impression that I might not get your first comment. So, re-opened this resolved conversation, and now it's clear what you meant by your first comment. 🦻 

Yup, PropsUtils is un-necessary now since we have marshal, and unmarshal in props interface already.